File: BaekJoon Online Judge/C++/Step10-Brute Force/1436.cpp
//
//  1436.cpp
//  Step10-Brute Force
//
//  Created by 김예빈 on 2019. 9. 5..
//  Copyright © 2019년 김예빈. All rights reserved.
//

#include <iostream>
using namespace std;

int main(int argc, const char * argv[]) {
    cin.tie(NULL);
    ios::sync_with_stdio(false);
    
    int n, result = 0, cnt = 666;
    
    cin >> n;
    
    while(1) {
        string str = to_string(cnt);
        
        if(str.find("666") != -1)
            result++;
        
        if(result == n) {
            cout << str;
            break;
        }
        
        cnt++;
    }

    return 0;
}

File: BaekJoon Online Judge/Swift/Step11-Brute Force/1436.swift
//
//  1436.swift
//  Step11-Brute Force
//
//  Created by Yebin Kim on 2019/11/29.
//  Copyright © 2019 김예빈. All rights reserved.
//

import Foundation

// Timeout
if let N = Int(readLine()!) {
    var cnt = 666
    var answer = 0
    
    while true {
        let str = String(cnt)
        
        if str.contains("666") {
            answer += 1
        }
        
        if answer == N {
            print(cnt)
            break
        }
        
        cnt += 1
    }
}
